internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/ControllerApiConstants.java

```diff
@@ -232,6 +232,7 @@ public class ControllerApiConstants {
   public static final String KAFKA_TOPIC_RETENTION_IN_MS = "kafka.topic.retention.in.ms";
   public static final String KAFKA_TOPIC_MIN_IN_SYNC_REPLICA = "kafka.topic.min.in.sync.replica";
   public static final String UPSTREAM_OFFSET = "upstream_offset";
```

Comment:
Could you help to double check if this field is still useful after this PR is in?